app/src/main/java/com/example/tungcon/demo/data/model/Movie.java

```diff
@@ -0,0 +1,145 @@
+    private String mTagline;
+    private String mCastList;
+    private String mImage;
+    private String mStory;
+
+    public Movie(JSONObject jsonObject) throws JSONException {
+            mYear = jsonObject.getInt(JSONKey.MOVIE_YEAR);
+            mDuration = jsonObject.getString(JSONKey.MOVIE_DURATION);
+            mMovie = jsonObject.getString(JSONKey.MOVIE_MOVIE);
+            mRating = (float) jsonObject.getDouble(JSONKey.MOVIE_RATING);
+            mStory = jsonObject.getString(JSONKey.MOVIE_STORY);
+            mCastList = Cast.getName(jsonObject.getString(JSONKey.MOVIE_CAST));
+            mTagline = jsonObject.getString(JSONKey.MOVIE_TAGLINE);
+            mImage = jsonObject.getString(JSONKey.MOVIE_IMAGE);
+            mDirDector = jsonObject.getString(JSONKey.MOVIE_DIRECTOR);
+    }
+
+    public int getYear() {
+        return mYear;
+    }
+
+    public void setYear(int year) {
+        mYear = year;
+    }
+
+    public String getMovie() {
+        return mMovie;
+    }
+
+    public void setMovie(String movie) {
+        mMovie = movie;
+    }
+
+    public float getRating() {
+        return mRating;
+    }
+
+    public void setRating(float rating) {
+        mRating = rating;
+    }
+
+    public String getDuration() {
+        return mDuration;
+    }
+
+    public void setDuration(String duration) {
+        mDuration = duration;
+    }
+
+    public String getDirDector() {
+        return mDirDector;
+    }
+
+    public void setDirDector(String dirDector) {
+        mDirDector = dirDector;
+    }
+
+    public String getTagline() {
+        return mTagline;
+    }
+
+    public void setTagline(String tagline) {
+        mTagline = tagline;
+    }
+
+    public String getCastList() {
+        return mCastList;
+    }
+
+    public void setCastList(String castList) {
+        mCastList = castList;
+    }
+
+    public String getImage() {
+        return mImage;
+    }
+
+    public void setImage(String image) {
+        mImage = image;
+    }
+
+    public String getStory() {
+        return mStory;
+    }
+
+    public void setStory(String story) {
+        mStory = story;
+    }
+
+    public static class Cast {
+        private String name;
+
+        public String getName() {
+            return name;
+        }
+
+        public void setName(String name) {
+            this.name = name;
+        }
+
+        public static String getName(String cast) {
+            String result = "";
+            try {
+                JSONArray array = new JSONArray(cast);
+                for (int i = 0; i < array.length(); i++) {
+                    JSONObject c = array.getJSONObject(i);
+                    result = result.concat(c.getString(JSONKey.MOVIE_NAME));
+                }
+            } catch (JSONException e) {
+                e.printStackTrace();
+            }
+            return result;
+        }
+    }
+
+    public static class JSONKey{
+        public static final String MOVIE_OBJECT = "movies";
```

Comment:
Em có thể remove tiền tố MOVIE_ vì nó là nested class của movie rồi 
```suggestion
        public static final String OBJECT = "movies";
```